target/linux/ath79/dts/ar9344_araknis_an-300-ap-i-n.dts

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9344.dtsi"
+#include "ar934x_senao_loader.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "araknis,an-300-ap-i-n", "qca,ar9344";
+	model = "Araknis AN-300-AP-I-N";
+
+	aliases {
+		label-mac-device = &eth0;
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "amber:power";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+			default-state = "off";
```

Comment:
Why should it be off in default state?

Reply:
for these boards, all LEDs are viewed through the same hole in the case. If power is on all the time it pretty much makes the rest of the LEDs useless. OEM only uses 1 LED for all functions and it's not this one. I call this LED power because it correlates to the power LED for other devices of exactly the same hardware (EAP600 / EAP1200 / EAP1750).

you can see the lightguide in FCC photos, first page

https://fccid.io/2AG6R-AN500APIAC/Internal-Photos/Internal-Photos-2917591.pdf

Reply:
Not related to the original question, but be aware that "off" is default and thus can be omitted if you want to keep the behavior.